### ISSUE TITLE
Inconsitency of EditSession.getLine

### ISSUE THREAD

**Issue by tomitnh** [ajaxorg/ace#2780]
It doesn't make sense to have EditSession.getLine(Number Row), while Editor.gotoLine( Number line ... )

**Comment by nightwing** [comment]
The argument names were different because row usually is 0 based index and gotoline argument is 1 based.
gotoLine probably needs to be deprecated as it is the only method using 1 based numbering.

**Comment by tomitnh** [comment]
Yeah, I suppose the root of the problem is row being 0-based-index and line being 1-based-index. Though as long as there is consistency within the madness, the user (developer) should be OK.

Could getLine's parameter being Number line? It would be more intuitive to give line-number when calling line-related functions.

**Comment by nightwing** [comment]
All the methods are 0 based, we can't switch to 1 based lines.

**Comment by tomitnh** [comment]
I am not sure what you mean by "all method are 0-based". Nonetheless, I am
just offering some solution to make your API consistent.

Consistency, consistency, consistency.
